src/main/resources/db/migration/general/V4_0__modify_notification_add_snooze.sql

```diff
@@ -0,0 +1,52 @@
+ALTER TABLE "SHIFT_NOTIFICATION"
+    ADD COLUMN ACKNOWLEDGED BOOLEAN DEFAULT FALSE NOT NULL;
+
+UPDATE "SHIFT_NOTIFICATION"
+SET "ACKNOWLEDGED" = TRUE
+WHERE "READ" = TRUE
+   OR "SENT_SMS" = TRUE
+   OR "SENT_EMAIL" = TRUE;
+
+ALTER TABLE "SHIFT_NOTIFICATION"
+    DROP COLUMN "READ";
+
+ALTER TABLE "SHIFT_NOTIFICATION"
+    DROP COLUMN "SENT_SMS";
+
+ALTER TABLE "SHIFT_NOTIFICATION"
+    DROP COLUMN "SENT_EMAIL";
+
+ALTER TABLE "SHIFT_NOTIFICATION"
+    DROP COLUMN "DATE_TIME";
```

Comment:
Out of curiousity, what was this column for. Was it for when the notification was created?

Reply:
It's the day the Sql was run, so in the query its just SELECT SYS_DATE, etc
https://github.com/ministryofjustice/cmd-csr-procs/blob/master/Notification_Shift/GET_MODIFIED_SHIFTS.sql#L52
It'll be to make notifications unique in the database, but its not an audit of changes, its a scratch database so I don't care if the shift changes twice before we get a chance to send the notificaiton, we only want to send the latest